src/component/component.yaml

```diff
@@ -0,0 +1,28 @@
+id: PCLCrypto
+name: Cryptography Plugin
+publisher: Andrew Arnott
+publisher-url: https://twitter.com/aarnott
+summary: PCL Storage provides a consistent, portable set of crypto APIs across .NET platforms.
+version: 1.1.18-g36fd0b2956
```

Comment:
Hmmm... I'll want to get this version to be programmatically supplied during the build rather than hard-coded in the source tree. Does this file allow parameterization or will I have to invent it?